docs/installation.md

```diff
@@ -0,0 +1,3 @@
+---
```

Comment:
If we have this document already, then you can put "Documentation" link in the navbar, which points to it 🙂

Reply:
Will be in next PR